raiden/network/transport/matrix.py

```diff
@@ -162,6 +162,7 @@ def _http_retry_delay() -> Iterable[float]:
                 log.warning(f"Selected server '{self._server_url}' not usable. Retrying.")
 
         self.greenlets = list()
+        self.event_start = gevent.Event()
```

Comment:
Please, do not add `event_start`. Checking for `__bool__` value of the `Runnable` should be enough

Reply:
this is still a WIP. event_start was added before your Runnable branch was merged.